.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -32,8 +32,9 @@
 
 # Testing
 <!--
-    Talk about the testing you have (and have not) done. It's useful
-    for others to know what you've already checked so that they can
+    Talk about the testing you have done, and the testing you have not done
+    whether you rely on automated tests or you don't know how to test something.
+    It's useful for others to know what you've already checked so that they can
     avoid repeating the same things and consider what you might have missed.
```

Comment:
What do you feel about expanding this a bit to:

```suggestion
    Talk about the testing you have done, and also specify any testing you
    know you have not done, because you rely on automated tests or
    because you just don't know how to test something.
    It's useful for others to know what you've already checked so that they can
    avoid repeating the same things and consider what you might have missed.
```

Reply:
It doesn’t seem to add help with anything but it rather lengthen the pull request template which makes it less practical to use.

Reply:
I just don't think the current sentence reads like fluent clear English, but my suggestion might be too long. Maybe @Aerozol or @mwiencek have better ideas as native speakers :)

Reply:
It is very possible, for example my previous comment just had two errors, and I have a tendency to compressing text, for no real benefit when it takes more time to decipher.

Reply:
how about
>  Talk about the testing you have or haven't done (whether you rely
    on automated tests, or don't know how to test something). It's useful
    for others to know what you've already tested so that they can avoid
    repeating the same steps, and instead consider what you might've
    missed.